src/backend/browser/fileImpl.ts

```diff
@@ -176,3 +179,44 @@ export const checkFileExistsImpl: (typeof window)[typeof SandboxKey]["checkFileE
 
     return Promise.resolve(fileEntries.includes(fileName));
   };
+
+// FileSystemFileHandleを保持するMap。キーは生成した疑似パス。
+const fileHandleMap: Map<string, FileSystemFileHandle> = new Map();
+
+// 疑似パスを返す
```

Comment:
メモがちょっとわかりにくいかもですね　こんな感じかな

```suggestion
// ファイル選択ダイアログを開く
// 返り値はファイルパスではなく、疑似パスを返す
```